tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/SharePointInfo/Search.cs

```diff
@@ -133,7 +133,7 @@ public SearchResponseVM GetMatters(SearchRequestVM searchRequestVM)
                     var managedColumns = configuration.GetSection("ContentTypes").GetSection("ManagedColumns").GetChildren();
                     foreach (var key in managedColumns)
                     {
-                        managedProperties.Add("MC" + key.Value);
```

Comment:
This looks good now. In future, we should have separating settings for managed properties and use it for search instead of directly using site columns.